subsys/bluetooth/audio/shell/bap.c

```diff
@@ -4245,9 +4245,9 @@ static ssize_t nonconnectable_ad_data_add(struct bt_data *data_array,
 		uint32_t broadcast_id;
 		int err;
 
-		err = bt_bap_broadcast_source_get_id(default_source.bap_source, &broadcast_id);
+		err = bt_rand(&broadcast_id, BT_AUDIO_BROADCAST_ID_SIZE);
 		if (err != 0) {
-			printk("Unable to get broadcast ID: %d\n", err);
+			printk("Unable to generate broadcast ID: %d\n", err);
```

Comment:
For testing purposes we might actually want to have a static ID as well, e.g. for testing with ID = 0x000000 as we've seen some platforms have. 

Since this value is only applied if we have created a broadcast source, we could add broadcast_id as as argument to `cmd_create_broadcast`. It is OK to keep as is (keeping existing implementation), but could be a nice followup